packages/nextjs/src/config/withSentryConfig.ts

```diff
@@ -115,19 +115,43 @@ function setUpTunnelRewriteRules(userNextConfig: NextConfigObject, tunnelPath: s
       destination: 'https://o:orgid.ingest.sentry.io/api/:projectid/envelope/?hsts=0',
     };
 
+    const tunnelRouteRewriteWithRegion = {
+      // Matched rewrite routes will look like the following: `[tunnelPath]?o=[orgid]&p=[projectid]?r=[region]`
+      // Nextjs will automatically convert `source` into a regex for us
+      source: `${tunnelPath}(/?)`,
+      has: [
+        {
+          type: 'query',
+          key: 'o', // short for orgId - we keep it short so matching is harder for ad-blockers
+          value: '(?<orgid>\\d*)',
+        },
+        {
+          type: 'query',
+          key: 'p', // short for projectId - we keep it short so matching is harder for ad-blockers
+          value: '(?<projectid>\\d*)',
+        },
+        {
+          type: 'query',
+          key: 'r', // short for region - we keep it short so matching is harder for ad-blockers
+          value: '(?<region>\\[a-z\\]{2})',
+        },
+      ],
+      destination: 'https://o:orgid.ingest.:projectid.sentry.io/api/:projectid/envelope/?hsts=0',
```

Comment:
typo?
```suggestion
      destination: 'https://o:orgid.ingest.:region.sentry.io/api/:projectid/envelope/?hsts=0',
```

Reply:
Wait how the hell did this work when I tried it out... Is our matcher in Sentry very lenient?